packages/dataclass-editor/src/data/type-data.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, expect } from 'vitest';
+import type { DataclassType, JavaType } from '../protocol/types';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { typeData } from './type-data';
+
+// Mock Data
+const mockDataClasses: DataclassType[] = [
+  { fullQualifiedName: 'com.example.ClassA', name: 'ClassA', packageName: 'com.example', path: '/path/ClassA' },
+  { fullQualifiedName: 'com.example.ClassB', name: 'ClassB', packageName: 'com.example', path: '/path/ClassB' }
+];
+
+const mockIvyTypes: JavaType[] = [
+  { fullQualifiedName: 'ivy.IvyTypeA', packageName: 'ivy', simpleName: 'IvyTypeA' },
+  { fullQualifiedName: 'com.example.TypeB', packageName: 'com.example', simpleName: 'TypeB' }
+];
+
+const mockNonIvyTypes: JavaType[] = [
+  { fullQualifiedName: 'com.other.TypeC', packageName: 'com.other', simpleName: 'TypeC' },
+  { fullQualifiedName: 'com.other.TypeD', packageName: 'com.other', simpleName: 'TypeD' }
+];
```

Comment:
do not understand why this is called "mock"
this has nothing to do with a mock. It is just test data...